lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -78,11 +79,12 @@ $(document).ready(function() {
   ShowMore.init();
 
   // Intialize takeover modal.
-  Takeover.init();
+  // Takeover.init();
 
   // Add fallback for HTML5 video on mobile tiles
   Tile.init();
 
+  let donation = new OrganDonation($('.takeover-container'));
```

Comment:
If the `OrganDonation` class isn't used outside the module, it might make sense to only export an `init` function from that module rather than exporting the class:

```js
class OrganDonation {
  // ...
}

export default {
  init($el = $('.takeover-container')) {
    new OrganDonation($el);
  }
}
```